### PR TITLE
nveto bug fix and add parameter

### PR DESCRIPTION
**Pull request by mzks**
**What is the problem / what does the code in this PR do**
 - Bug fix to load nVeto config
 - Add a PMT parameter for nVeto (absorption efficiency)
 - Rename several parameters for https://github.com/XENONnT/private_nt_aux_files/pull/45

### CODE REVIEW

File: wfsim/load_resource.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def __init__(self, config=None):
             if config['enable_pmt_afterpulses']:
                 self.uniform_to_pmt_ap = straxen.get_resource(files['photon_ap_cdfs'], fmt='pkl.gz')
 
-        if config['detector'] == 'XENONnT':
+        if config['detector'] == 'XENONnT' and not config['neutron_veto']:
```
Comment (ramirezdiego) [inline wfsim/load_resource.py:136]: @mzks Why is this change necessary?

**Comment by ramirezdiego** [comment]
This branch slightly overlaps with the fixes in https://github.com/XENONnT/WFSim/pull/125. Let's please review and merge https://github.com/XENONnT/WFSim/pull/125 first (and as soon as possible).

**Comment by mzks** [comment]
The changes moved to #125
